File: src/main/lesson7/javahw/Month.java
package javahw;

public enum Month {
    JANUARY,
    FEBRUARY,
    MARCH,
    APRIL,
    MAY,
    JUNE,
    JULY,
    AUGUST,
    SEPTEMBER,
    OCTOBER,
    NOVEMBER,
    DECEMBER;

    public boolean isWinter() {
        return this == DECEMBER || this == JANUARY || this == FEBRUARY;
    }
}

File: src/main/lesson7/kotlinhw/Month.kt
package kotlinhw

enum class Month(val isWinterMonth: Boolean) {
    JANUARY(true),
    FEBRUARY(true),
    MARCH(false),
    APRIL(false),
    MAY(false),
    JUNE(false),
    JULY(false),
    AUGUST(false),
    SEPTEMBER(false),
    OCTOBER(false),
    NOVEMBER(false),
    DECEMBER(true);

    fun isWinter() = isWinterMonth
}